### PR TITLE
fix: realpath on mac

### PR DESCRIPTION
**Pull request by jLopezbarb**
# Proposed changes

DEV-122

Create a new function to check the real name of the folders. This new code doesn't resolve symlinks, it just replaces the path name on OS with no sensitive paths 

## How to validate

1. Clone okteto/go-getting-started
1. Run cd go-Getting-StartEd
1. Run okteto up
1. Update some code from main locally and check that is updated on the remote

## CLI Quality Reminders 🔧

For both authors and reviewers:

- Scrutinize for potential regressions
- Ensure key automated tests are in place
- Build the CLI and test using the validation steps
- Assess Developer Experience impact (log messages, performances, etc)
- If too broad, consider breaking into smaller PRs
- Adhere to our [code style](https://github.com/okteto/okteto/blob/master/docs/code-style.md) and [code review](https://github.com/okteto/okteto/blob/master/docs/code-review.md) guidelines

### CODE REVIEW

**Review by ifbyol — APPROVED** [review]
Linters are failing, besides that, changes look good and work fine!

File: pkg/model/dev.go [hunk 1]
```diff
@@ -282,50 +284,64 @@ func NewDev() *Dev {
...
 	for i := range dev.Volumes {
 		if dev.Volumes[i].LocalPath == "" {
 			continue
 		}
-		dev.Volumes[i].LocalPath = loadAbsPath(folder, dev.Volumes[i].LocalPath)
+		dev.Volumes[i].LocalPath = loadAbsPath(folder, dev.Volumes[i].LocalPath, fs)
 	}
 	for i := range dev.Sync.Folders {
-		dev.Sync.Folders[i].LocalPath = loadAbsPath(folder, dev.Sync.Folders[i].LocalPath)
+		oktetoLog.Println("folder", dev.Sync.Folders[i].LocalPath)
```
Comment (andreafalzetti) [inline pkg/model/dev.go:322]:
```suggestion
```

File: pkg/model/dev.go [hunk 2]
```diff
@@ -282,50 +284,64 @@ func NewDev() *Dev {
...
+		dev.Sync.Folders[i].LocalPath = loadAbsPath(folder, dev.Sync.Folders[i].LocalPath, fs)
+		oktetoLog.Println("folder", dev.Sync.Folders[i].LocalPath)
```
Comment (andreafalzetti) [inline pkg/model/dev.go:324]:
```suggestion
```

**Review by andreafalzetti — APPROVED** [review]
tested and works as expected ✅

**Pull request merged**